index.js

```diff
@@ -118,8 +136,8 @@ function getLastCarInfo(/* code here */) {
  * For example, if getCarInfoById is invoked with the inventory and the number 1,
  * it will return `This is a Lincoln Navigator`.
 */
-function getCarInfoById(/* code here */) {
-  /* code here */
+function getCarInfoById(carsArray, id) {
+  return 'This is a ' + carsArray[id-1].car_make + carsArray[id-1].car_model;
 }
 
```

Comment:
For lines 139 to 140, the test probably passed, however if i were to remove an item from this array then your function would not work properly any more because the ids and index won't match as nicely.  A better way to do this (without using the .filter method) would be to use an if statement inside a for loop.  The for loop tells the function to loop through the array (and give you the index which is what we needed). Then the if statement will compare the id of each object to the id that you passed in.  Finally the if statement will only return this statement if the ids match.